### ISSUE TITLE
How do you disable the plugin?

### ISSUE THREAD

**Issue by veteri** [truong-bui/AsyncLoadingScreen#89]
Disabling the entire plugins isnt enough and makes loading a level freeze for as long was set in the "Min load time" setting...

**Comment by truong-bui** [comment]
Hi, if you use the Marketplace version, you disable it in the Plugin in the editor. If you use the GitHub version, you need to delete it in your projects' Plugin folder. If you want to temporarily disable the loading screen at a specific map, use those [nodes ](https://github.com/truong-bui/AsyncLoadingScreen?tab=readme-ov-file#version-140-2512023) before Open Level.

**Comment by veteri** [comment]
Yeah i was using the marketplace version. I'm not sure what was going on but disabling the plugin didn't actually disable it until i worked on the project the next day again.

**Comment by truong-bui** [comment]
You need to restart the editor after disabling/enabling any plugins.

**Comment by flo3806** [comment]
Hi, i got the same problem, i ve installed this plugin from the marketplace and i tryied it with 5sec timer, but it seems to be stucked on 5 seconds, everytime, even if i change the value, even with -1, everytime i got 5 seconds between every level loading. And i tryied to remove it from the engine, and uninstall and reinstall the engine, that does the same, there is a probleme, my project is broken now cause of this plugin. Did you find a solution ?
